src/xcode/ENA/ENA/Source/Services/TicketValidation/Models/TicketValidationInitializationData.swift

```diff
@@ -13,4 +13,26 @@ struct TicketValidationInitializationData: Codable {
 	let consent: String
 	let subject: String
 	let serviceProvider: String
+	
+	static func fake (
```

Comment:
```suggestion
	static func fake(
```

Also, I think you can move this to an extension in a separate file and for now only include it in the test target so we don't accidentally use this in production.

Reply:
done